### DISCUSSION TITLE
categories/%EA%B0%9C%EB%B0%9C/%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC/WebSocket-%EC%B1%95%ED%84%B0-1-%EC%84%9C%EB%A1%A0%EA%B3%BC-%EC%9B%B9%EC%86%8C%EC%BC%93-API/

### DISCUSSION THREAD

**Discussion by giscus** [redniche/redniche.github.io#6] [Announcements]
# categories/%EA%B0%9C%EB%B0%9C/%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC/WebSocket-%EC%B1%95%ED%84%B0-1-%EC%84%9C%EB%A1%A0%EA%B3%BC-%EC%9B%B9%EC%86%8C%EC%BC%93-API/

웹소켓(WebSocket) 서론

https://yechankun.github.io/categories/%EA%B0%9C%EB%B0%9C/%EB%84%A4%ED%8A%B8%EC%9B%8C%ED%81%AC/WebSocket-%EC%B1%95%ED%84%B0-1-%EC%84%9C%EB%A1%A0%EA%B3%BC-%EC%9B%B9%EC%86%8C%EC%BC%93-API/

**Comment by yechankun** [comment]
https://hpbn.co/websocket/#websocket-api
의 내용을 번역한 것입니다.
